### DISCUSSION TITLE
failover / backup connection

### DISCUSSION THREAD

**Discussion by dvinik** [qdm12/gluetun#1347] [Q&A]
Hi,

Is there a way to set up a failover (backup) connection for gluetun to switch over to when the primary connection is "unhealthy" (dropped, disconnected, slow or error) ?

**Comment by qdm12** [comment]
(Copy pasting from email)

For now if it gets unhealthy, it'll restart the vpn using a server picked at random from the pool of servers depending on your filters. So you could set SERVER_REGIONS=abc,xyz so it picks one of the 2 regions at random.
There is an issue on github https://github.com/qdm12/gluetun/issues/290 to implement a rotation between them, I guess that would fit your needs?

**Comment by dvinik** [comment]
Thank you!



Best regards,

David Vinik

On Jan 14, 2023, at 4:38 PM, Quentin McGaw ***@***.***> wrote:

﻿

(Copy pasting from email)

For now if it gets unhealthy, it'll restart the vpn using a server picked at random from the pool of servers depending on your filters. So you could set SERVER_REGIONS=abc,xyz so it picks one of the 2 regions at random.
There is an issue on github #290<https://github.com/qdm12/gluetun/issues/290> to implement a rotation between them, I guess that would fit your needs?
